great_expectations/rule_based_profiler/parameter_builder/parameter_builder.py

```diff
@@ -420,11 +420,7 @@ def get_metrics(
 
         # Nineth: Compose and return result to receiver (apply simplications to cases of single "metric_value_kwargs").
         return MetricComputationResult(
-            metric_values=list(attributed_resolved_metrics_map.values())[
-                0
-            ].metric_values
-            if len(metric_value_kwargs) == 1
-            else attributed_resolved_metrics_map.values(),
+            list(attributed_resolved_metrics_map.values()),
```

Comment:
:heart:

Reply:
This is a long method, it seems we can encapsulate much of what is happening here in MetricValues (have MetricValues responsible for itself rather than having ParameterBuilder manipulate it)

Reply:
@donaldheppner These types are placeholder ones, which we intended to replace as part of a MAINTENANCE task (I just created a JIRA item for it).  Thanks.